src/lib.rs

```diff
@@ -1140,8 +1140,9 @@ impl<T: Activated + ?Sized> Capture<T> {
         }
     }
 
-    /// Adds a filter to the capture using the given BPF program string. Internally
-    /// this is compiled using `pcap_compile()`.
+    /// Sets the pcap capture filter using the given BPF program string. Internally
+    /// this is compiled using `pcap_compile()`. `optimize` specifies controls whether
```

Comment:
Small typo: "`optimize` specifies controls". I guess you should just pick one: either "specifies" or "controls" and then I'll merge.